ibis/backends/duckdb/__init__.py

```diff
@@ -208,6 +209,55 @@ def _register_failure(self):
             f"please call one of {msg} directly"
         )
 
+    def read_json(
+        self,
+        source_list: str | list[str] | tuple[str],
+        table_name: str | None = None,
+        **kwargs,
+    ) -> ir.Table:
+        """Read newline-delimited JSON into an ibis table.
+
+        Parameters
+        ----------
+        source_list
+            File or list of files
+        table_name
+            Optional table name
+        kwargs
+            Additional keyword arguments passed to DuckDB's `read_json_objects` function
```

Comment:
Does `read_json_objects` take any keyword arguments?  I can't find any.

Reply:
It doesn't :)

Reply:
Should I take this out or leave it for some future where there may or may not be kwargs?

Reply:
I suppose it isn't doing any harm where it is, and if they add kwargs in the future it _may_ "just work"